ta4j-core/src/main/java/org/ta4j/core/AnalysisCriterion.java

```diff
@@ -39,6 +39,14 @@
  */
 public interface AnalysisCriterion {
 
+    /** Filter to differentiate between winning and losing positions. */
+    public enum PositionFilter {
```

Comment:
This enum should be moved to its own file. It is used by some criterias

Reply:
Can we find a better name? (Position)ReturnType?

Reply:
- PositionType (not good, this it irritating with Buy and Sell)
- PositionSection
- PositionArea
- PositionItem
- PositionPart (good; means: either the profit part or the loss part)
- PositionReturnType (good, but just a little long name when using in so many criterions)

@team172011 You can choose.

Reply:
I selected `PositionPart`, but if you find a better name, we can change that.

`@param positionPart the PositionPart to select either profit or loss positions`

The benefit is that the name is not only short, but it can be used also for other classes not only limited to a `returnType`.